File: FoodDlvAPI/Services/ProductSelectionService.cs
using FoodDlvAPI.DTOs;
using FoodDlvAPI.Interfaces;
using FoodDlvAPI.Models;

namespace FoodDlvAPI.Services
{
    public class ProductSelectionService
    {
        //Fields        
        private readonly IProductSelectionRepository _repository; 

        //Constuctors
        public ProductSelectionService(IProductSelectionRepository repository)
        {
            _repository = repository;
        }
                
        public ProductDTO ProductSelection(int productId, bool? status)
        {
            var data = _repository.GetProductSelection(productId, status);

            return data;
        }
    }
}

File: food-dlv-website/src/services/Cart/productSelection.service.js
import axios from 'axios';

const ProductSelection_API_URL = 'https://localhost:7093/api/ProductSelection/';

class ProductSelectionService{
    getProductSelect(productId,state){
        return axios.get(          
            `${ProductSelection_API_URL}?productId=${productId}&status=${state}`
        );
    }}

export default new ProductSelectionService();